[2023-03-17T04:35:57.464+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Monthly.extract2 scheduled__2023-02-24T00:00:00+00:00 [queued]>
[2023-03-17T04:35:57.487+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Monthly.extract2 scheduled__2023-02-24T00:00:00+00:00 [queued]>
[2023-03-17T04:35:57.487+0530] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-03-17T04:35:57.487+0530] {taskinstance.py:1284} INFO - Starting attempt 2 of 4
[2023-03-17T04:35:57.487+0530] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-03-17T04:35:57.508+0530] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract2> on 2023-02-24 00:00:00+00:00
[2023-03-17T04:35:57.517+0530] {standard_task_runner.py:55} INFO - Started process 4658 to run task
[2023-03-17T04:35:57.527+0530] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Monthly', 'extract2', 'scheduled__2023-02-24T00:00:00+00:00', '--job-id', '4875', '--raw', '--subdir', 'DAGS_FOLDER/Monthly.py', '--cfg-path', '/tmp/tmpx5igvx6c']
[2023-03-17T04:35:57.534+0530] {standard_task_runner.py:83} INFO - Job 4875: Subtask extract2
[2023-03-17T04:35:57.635+0530] {task_command.py:389} INFO - Running <TaskInstance: Monthly.extract2 scheduled__2023-02-24T00:00:00+00:00 [running]> on host DESKTOP-SQIOQPD.
[2023-03-17T04:35:57.875+0530] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=nikheleshbhattacharyya@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Monthly
AIRFLOW_CTX_TASK_ID=extract2
AIRFLOW_CTX_EXECUTION_DATE=2023-02-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-24T00:00:00+00:00
[2023-03-17T04:36:01.638+0530] {logging_mixin.py:137} INFO - document.getElementById('2020').click()
[2023-03-17T04:36:01.721+0530] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/root/airflow/dags/Monthly.py", line 524, in extract2
    driver.execute_script(k)
  File "/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py", line 500, in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
  File "/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py", line 440, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/errorhandler.py", line 245, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.JavascriptException: Message: javascript error: Cannot read properties of null (reading 'click')
  (Session info: headless chrome=109.0.5414.119)
Stacktrace:
#0 0x55a9935bc303 <unknown>
#1 0x55a993390d37 <unknown>
#2 0x55a993394608 <unknown>
#3 0x55a9933943d2 <unknown>
#4 0x55a99339503c <unknown>
#5 0x55a9934063ae <unknown>
#6 0x55a9933ed982 <unknown>
#7 0x55a99340588c <unknown>
#8 0x55a9933ed753 <unknown>
#9 0x55a9933c0a14 <unknown>
#10 0x55a9933c1b7e <unknown>
#11 0x55a99360b32e <unknown>
#12 0x55a99360ec0e <unknown>
#13 0x55a9935f1610 <unknown>
#14 0x55a99360fc23 <unknown>
#15 0x55a9935e3545 <unknown>
#16 0x55a9936306a8 <unknown>
#17 0x55a993630836 <unknown>
#18 0x55a99364bd13 <unknown>
#19 0x7f02c4586609 start_thread

[2023-03-17T04:36:01.752+0530] {taskinstance.py:1322} INFO - Marking task as UP_FOR_RETRY. dag_id=Monthly, task_id=extract2, execution_date=20230224T000000, start_date=20230316T230557, end_date=20230316T230601
[2023-03-17T04:36:01.783+0530] {standard_task_runner.py:100} ERROR - Failed to execute job 4875 for task extract2 (Message: javascript error: Cannot read properties of null (reading 'click')
  (Session info: headless chrome=109.0.5414.119)
Stacktrace:
#0 0x55a9935bc303 <unknown>
#1 0x55a993390d37 <unknown>
#2 0x55a993394608 <unknown>
#3 0x55a9933943d2 <unknown>
#4 0x55a99339503c <unknown>
#5 0x55a9934063ae <unknown>
#6 0x55a9933ed982 <unknown>
#7 0x55a99340588c <unknown>
#8 0x55a9933ed753 <unknown>
#9 0x55a9933c0a14 <unknown>
#10 0x55a9933c1b7e <unknown>
#11 0x55a99360b32e <unknown>
#12 0x55a99360ec0e <unknown>
#13 0x55a9935f1610 <unknown>
#14 0x55a99360fc23 <unknown>
#15 0x55a9935e3545 <unknown>
#16 0x55a9936306a8 <unknown>
#17 0x55a993630836 <unknown>
#18 0x55a99364bd13 <unknown>
#19 0x7f02c4586609 start_thread
; 4658)
[2023-03-17T04:36:01.818+0530] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-03-17T04:36:01.849+0530] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
